### ISSUE TITLE
Incorrect ordering of outputs

### ISSUE THREAD

**Issue by snowypowers** [CityOfZion/neo-scan#107]
The outputs on neoscan is ordered incorrectly.

Doing a POST to a NEO node  for tx `60e2711319c1453d8de0ca8268b6ba9ced1d0c0c581c49d26e6efd4b75dccb24` yields: 
```
{
        "vout": [
            {
                "n": 0,
                "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
                "value": "0.1",
                "address": "ALq7AWrhAueN6mJNqk6FHJjnsEoPRytLdW"
            },
            {
                "n": 1,
                "asset": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
                "value": "0.005554",
                "address": "ALq7AWrhAueN6mJNqk6FHJjnsEoPRytLdW"
            }
        ],
...
}
```

However, on neoscan, it is displayed as:

![image](https://user-images.githubusercontent.com/9101245/34091473-ae4684e2-e372-11e7-8b9e-0fc4321bd06a.png)

This is not important to a normal end-user as the order would not matter to them. However, on the technical side, the coins are referenced by their *txid* and their *index* in this `vout` list. Thus, it would be good for it to be in the order specified for this to be useful for devs to look at.

Similarly, this could be done for the inputs. (so far I havnt have a use for that but consistency is nice)

TLDR: Please display the outputs as they are ordered in the transaction `vout` as it is important to the developer.

**Comment by snowypowers** [comment]
For clarity, this [tx](https://neoscan-testnet.io/transaction/2d32bad56d749c008499b2a723eaaced9fa980004b36e9a4d4fa00c85f6d0d25) shows that there is some ordering involved but definitely not according to the tx outputs. 

I guess we can either: 
1. Close as `willnotfix` as this is more consumer-focused, grouping by address does indeed help.
2. Sort by index to be more dev-focused. Or implement a toggle for that.
3. Implement some sort of script that would dump the item as an TransactionInput to the clipboard when clicked on.(actually this would be super cool then we can construct tx by using neoscan as a visual tool).

**Comment by lucianoengel** [comment]
I think we should keep the correct index. Will work on that and have a look on how to add  the 3. :)

**Comment by lucianoengel** [comment]
fixed the ordering
